local/gugcat/form.php

```diff
@@ -0,0 +1,66 @@
+  
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * My Media main viewing page.
+ *
+ * @package    local_gugcat
+ * @copyright  2020
+ * @author     Accenture
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+require_once(__DIR__ . '/../../config.php');
+require_once($CFG->dirroot.'/local/gugcat/classes/form/form.php');
+$PAGE->navbar->ignore_active();
+$PAGE->navbar->add(get_string('navname', 'local_gugcat'), new moodle_url('/local/gugcat'));
+
+$courseid = optional_param('id', 1, PARAM_INT);
```

Comment:
Maybe we can add a comment here to indicate that this is temporary while we are still working on the navigation to pass the course id.